File: JS_Problems/Cats_and_a_Mouse.js
// Complete the catAndMouse function below.
function catAndMouse(x, y, z) {
  if (Math.abs(z - x) === Math.abs(z - y)) {
    return "Mouse C";
  } else if (Math.abs(z - x) > Math.abs(z - y)) {
    return "Cat B";
  } else {
    return "Cat A";
  }
}

File: Python_Problems/Cats_and_a_Mouse.py
# Complete the catAndMouse function below.
def catAndMouse(x, y, z):
    if abs(z-y) == abs(z-x):
        return ("Mouse C")
    if abs(z-y) > abs(z-x):
        return ("Cat A")
    else:
        return ("Cat B")
